src/pages/workspaces/workspace/Data.js

```diff
@@ -1532,3 +1532,8 @@ export const navPaths = [
     title: ({ name }) => `${name} - Data`,
   },
 ];
+
+// These modules are exported for testing only.
+export const testOnlyExports = {
```

Comment:
For application code, I don't think we care enough about curating the exports / interface to distinguish these as test only. I would vote to simply export the component where it's defined.

```js
export const WorkspaceData = ...
```